### PR TITLE
Tantares configs overhaul, part 1

### PR DESCRIPTION
**Pull request by Niemand303**
I leave for a week, so that what I've done would already be in the mod in case a release happens.

### CODE REVIEW

**Comment by stratochief66** [comment]
@Niemand303 hey, sorry for the delay. There are a lot of changes in here. Some I find a little hard to follow, but nothing objectionable. Do you want to take another quick look at this before I merge it?

**Comment by Niemand303** [comment]
Sorry, I won't have time since I leave for France tonight (in 5 hours) :c
You can merge it, however, since I haven't made much progress since my return back to Moscow.
